reference/v8.json

```diff
@@ -1440,11 +1585,15 @@
       "function": "piecewise-constant",
       "zoom-function": true,
       "property-function": true,
-      "values": [
-        "map",
-        "viewport"
-      ],
-      "doc": "Control whether the translation is relative to the map (north) or viewport (screen)",
+      "values": {
+        "map": {
+            "doc": "The fill is translated relative to the map (north)."
+        },
+        "viewport": {
+            "doc": "The fill is translated relative to the viewport (screen)."
```

Comment:
What does “screen” mean here? Does it mean that a higher y value moves the fill’s origin up? Or down? (I suppose specifying one or the other would be problematic for platform-specific documentation such as mapbox/mapbox-gl-native#6066, but that isn’t the style specification’s problem.)